tests/Makefile

```diff
@@ -2,12 +2,9 @@
 MAKEFILE_DIR:=$(dir $(realpath $(firstword ${MAKEFILE_LIST})))
 REPO_ROOT:=$(realpath ${MAKEFILE_DIR}../)
 
-# Initcontainer language under test
-INITCONTAINER_LANGUAGE:=${INITCONTAINER_LANGUAGE}
```

Comment:
Should this be removed? It looks like the environment variable is referenced further down in the Makefile.